src/backend/distributed/commands/multi_copy.c

```diff
@@ -986,6 +1023,50 @@ OpenCopyTransactions(CopyStmt *copyStatement, ShardConnections *shardConnections
 
 
 /*
+ * BinaryCopyFormat iterates over columns of the relation given in rowOutputState and
+ * looks for a column whose type is array of user-defined type. If it founds that
```

Comment:
"If it founds" -> "If it finds one"

"and it return false" -> "and it returns"